### PR TITLE
Newbodies added to bodies module

### PR DESCRIPTION
**Pull request by abhijeetmanhas**
Addresses #234 
I have added class for moon and other planets of solar system + Pluto to `bodies.py` .I have used all data from [NASA JPL](https://ssd.jpl.nasa.gov/?planet_phys_par).
@shreyasbapat @ritzvik please review it

### CODE REVIEW

**Comment by abhijeetmanhas** [comment]
Should I write tests for it too before merging?

**Comment by rishi-s8** [comment]
@abhijeetmanhas indeed. Tests are important. :)

File: src/einsteinpy/bodies.py [hunk 1]
```diff
@@ -95,3 +95,111 @@ def __init__(self):
...
+class _Neptune(Body):
+    def __init__(self):
+        parent = Sun
+        name = "Neptune"
+        R = 24622 * u.km
+        mass = 1.024126e26 * u.kg
+        super(_Neptune, self).__init__(name=name, mass=mass, R=R, parent=parent)
+
+
+Neptune = _Neptune
```
Comment (ritzvik) [inline src/einsteinpy/bodies.py]: `Neptune = _Neptune()` ?

**Comment by ritzvik** [comment]
@shreyasbapat 

I've not developed the bodies module, but still, here's my two cents.

  - It wasn't a wise design decision to have `BlahBlahBody = _BlahBlahBody()` because each time we import anything from `bodies` module, all 11 `__init__()` in all predefined bodies are getting executed, making the module slow! 
  - Further, we are not giving user the freedom to make different instances of the same class. We are giving them predefined instances. Now it may be possible that the user might want 2 different instances of `_Sun()`, one with a slight change in it, but he won't be able to do that easily.

**Comment by shreyasbapat** [comment]
Well, it was never intended to have solar system bodies

**Comment by abhijeetmanhas** [comment]
It was mentioned in comments of `bodies.py` that it was supposed to have predefined bodies, I added those. I also thought that writing `body = _body` will make it slow, but I found it was done for sun and earth, so I followed same format.
So should I remove all lines containing `body = _body` even for earth and sun? Or should I not create different classes for other planets, but store their attributes in some dictionary and  create 3 parent classes: star, planet & satellite?

File: src/einsteinpy/bodies.py (hunk 1 above)
Comment (ritzvik) [reply src/einsteinpy/bodies.py]: Can you make this change and afterwards, rebase the branch?
@abhijeetmanhas

**Pull request closed without merging**